src/components/pages/address-overview/address-overview.tsx

```diff
@@ -30,7 +30,9 @@ const AddressOverview: React.FC<{ address?: Address }> = ({ address }) => {
       <Box>
         {address ? (
           <>
-            <CategoryName>{capitalizeFirstLetter(address.kind)}</CategoryName>
+            {address.kind && (
```

Comment:
This is mandatory because we can create addresses directly with Profile calls and if the payload contains a `null` `kind` field it crashes.